core/src/main/java/org/bitcoinj/script/ScriptBuilder.java

```diff
@@ -457,14 +476,18 @@ public static Script createP2PKOutputScript(ECKey pubKey) {
      * Creates a scriptPubKey that sends to the given public key hash.
      */
     public static Script createP2PKHOutputScript(byte[] hash) {
+        return new ScriptBuilder().p2pkhOutputScript(hash).build();
+    }
+
+    private ScriptBuilder p2pkhOutputScript(byte[] hash) {
         checkArgument(hash.length == LegacyAddress.LENGTH);
-        ScriptBuilder builder = new ScriptBuilder();
-        builder.op(OP_DUP);
-        builder.op(OP_HASH160);
-        builder.data(hash);
-        builder.op(OP_EQUALVERIFY);
-        builder.op(OP_CHECKSIG);
-        return builder.build();
+        checkState(chunks.isEmpty());
+        op(OP_DUP);
+        op(OP_HASH160);
+        data(hash);
+        op(OP_EQUALVERIFY);
+        op(OP_CHECKSIG);
+        return this;
```

Comment:
Can't this be:
```
    return this.op(OP_DUP)
                     .op(OP_HASH160)
                     ...
                     .op(OP_CHECKSIG);
```   
?

Reply:
Yes, ok, I  changed this for the methods I touched.